.github/workflows/binary.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - run: bun i --frozen-lockfile
       - run: git reset --hard
 
-      - uses: goreleaser/goreleaser-action@v4
+      - uses: goreleaser/goreleaser-action@v6
```

Comment:
I can revert this or move it to another PR if you want.

Seems safe to upgrade:

```
❯ goreleaser check
  • checking                                 path=.goreleaser.yml
  • 1 configuration file(s) validated
  • thanks for using goreleaser!
```

Reply:
See https://goreleaser.com/deprecations/